python/pyspark/sql/tests/connect/client/test_artifact.py

```diff
@@ -271,6 +271,21 @@ def func(x):
             self.spark.addArtifacts(f"{archive_path}.zip#my_files", archive=True)
             self.assertEqual(self.spark.range(1).select(func("id")).first()[0], "hello world!")
 
+    def test_copy_from_local_to_FS(self):
```

Comment:
Please add a server/scala-side test as well in `ArtifactManagerSuite`

Reply:
Sure.